Functions.Templates/Templates-v2/HttpTrigger-Python/function_body.py

```diff
@@ -1,5 +1,5 @@
 @app.function_name(name="$(FUNCTION_NAME_INPUT)")
-@app.route(route="hello")
+@app.route(route="$(ROUTE_VALUE)")
```

Comment:
Do you want the `route` parameter to also take user input in the `Create New Project with HttpTrigger function` job?

Reply:
I think it might be a good idea to use function name `FUNCTION_NAME_INPUT` for route, as it would be unique and would correlate with function as well. Also it would match the current templating experience as well as would be less confusing for a completely new user as they may not be familiar with functions.